src/lib/config.js

```diff
@@ -32,29 +32,35 @@ async function config() {
     return;
   }
 
-  // Build table of existing networks found in their config.json
+  // Checks if developer has the legacy networks in config.json and renames it to deploy aliases.
+  if (Object.prototype.hasOwnProperty.call(config, 'networks')) {
+    Object.assign(config, { deployAliases: config.networks });
+    delete config['networks'];
```

Comment:
This can be executed in one line. I chose to do it in two steps for readability.

```ts
delete Object.assign(config, { deployAliases: config.networks })['networks'];
```

Reply:
+1 for 2 lines. But since we're talking style, `delete config.networks;` because we're not passing in a var